Fixed the failed import of MockChannel

JamesDavidCarr opened the pull request:
@raulcf This was causing the project to fail to compile

The only fix that's required is build.gradle but I removed trailing whitespaces and optimised imports on the failing test, which I'm happy to remove if you want.

raulcf commented:
Looks good. A couple of comments:
- Never use wildcards in import (import x.y.*), instead include all that is necessary individually.
- See if you can make the commit message a bit more descriptive (it does not matter if it's longer).

Thanks,

JamesDavidCarr commented:
I realised what the issue was so the fix is now much smaller.

And I decided not to include the code cleanup in this PR since it's actually a separate issue.

Commit:
Added missing dependencies to seep-integrationtests

The contents of the seep-integrationtests module were not loading
because the tests are present in the src/ package and gradle was adding
dependencies to the test/ package.

raulcf commented:
Looks good, thanks!
